pp/peers/pp.go

```diff
@@ -94,3 +98,28 @@ func NewServer(ctx context.Context) *PPServer {
 
 	return ppServer
 }
+
+func ListenSendPacket(handler func(core.WritePacketCostTime)) {
+	quit := make(chan os.Signal, 1)
```

Comment:
don't add os.exit(1) out of nowhere

Reply:
working in [qb-1373 add proper closer functions for the pp node] to re-struct the server close